AppManager/app_manager_server.py

```diff
@@ -501,7 +478,9 @@ def unmonitor_and_terminate(watch):
     watch: A string specifying the Monit entry.
   """
   try:
-    unmonitor(watch)
+    monit_retry = retry(max_retries=5, retry_on_exception=MonitUnavailable)
```

Comment:
Default backoff sequence is 0.2, 0.4, 0.8, 1.6, 3.2, 6.4, 12.8,... If you set `max_retries=5` sum of pauses between retries will be `6.2s`. Is this usually enough for monit to become available again?
`retrying_timeout=30` or 60 could be an alternative.